assets/javascripts/src/components/InputField.jsx

```diff
@@ -4,13 +4,14 @@ export class AmountInput extends React.Component {
     render() {
         const { inputAmount, symbol, refFn, ...props } = this.props;
 
-        return <span className="contribute-controls__input contribute-controls__input--amount input-text">
+        return <span className={(this.props.small ? 'contribute-controls__input-small':'contribute-controls__input') + ' contribute-controls__input--amount input-text'}>
```

Comment:
I would conditionally add an extra class here, `contribute-controls__input--amount--small`, leaving the classes as before if `props.small` is false.